lib/elixir/lib/gen_server.ex

```diff
@@ -159,6 +159,36 @@ defmodule GenServer do
   the GenServer callbacks as doing so will cause the GenServer to misbehave.
   If you want to receive custom messages, always receive them in `handle_info/2`.
 
+  Custom messages are all other messages a server may receive that are not sent via
+  `call/3` or `cast/2`.
+
+  To send custom message use: `Kernel.send/2`,`Process.send_after/3` functions.
```

Comment:
I see the need for expressivity, but I still feel that level of expressivity doesn't belong to the docs for `GenServer` (e.g., we shouldn't explain how to send messages in GenServer, but ensure users know where to look from GenServer if they want to know more about sending messages). 😕 Also, I feel it's dangerous to mention `Process.send_after/3` alongside `Kernel.send/2` because the two have different importance in the Elixir landscape and the second one is tricky to use correctly without the possibility of causing harm.

I'd put this info in parens here because we're not documenting how to send messages, just pointing users to where they could learn more (again, because we're in the docs for GenServer). I would love other people's opinion here though :)